src/app/legacy/containers/OnDemandImage/index.jsx

```diff
@@ -21,7 +21,7 @@ const ImageContainer = styled.div`
 `;
 
 const getSrc = ({ imageUrl, size }) =>
-  `https://${imageUrl.replace('$recipe', `${size}x${size}`)}`;
+  `https://${imageUrl.replace('$recipe', `${size}x${size}`)}.webp`;
```

Comment:
what if the ondemand data comes back with a .webp image? Is that a risk that we will end up with a `.webp.webp` src?